3-enrich/scala-common-enrich/src/main/scala/com.snowplowanalytics.snowplow.enrich/common/enrichments/registry/AnonIpEnrichment.scala

```diff
@@ -77,22 +87,61 @@ object AnonOctets extends Enumeration {
 
   /**
    * Convert a Stringly-typed integer
-   * into the corresponding AnonOctets
+   * into the corresponding AnonIPv4Octets
    * Enum Value.
    *
    * Update the Validation Error if the
    * conversion isn't possible.
    *
-   * @param anonOctets A String holding
+   * @param anonIPv4Octets A String holding
    *        the number of IP address
    *        octets to anonymize
-   * @return a Validation-boxed AnonOctets
+   * @return a Validation-boxed AnonIPv4Octets
    */
-  def fromInt(anonOctets: Int): ValidatedMessage[AnonOctets] =
+  def fromInt(anonIPv4Octets: Int): ValidatedMessage[AnonIPv4Octets] =
```

Comment:
Technically the type is `ValidatedMessage[_] \/ Throwable`. Maybe it's better to have it fully `ValidatedMessage` and return a generic failure.

Reply:
Totally agree, but similar code is in all enrichments, so I wanted to keep it consistent.